spec/gcov/gcov_test_cases_spec.rb

```diff
@@ -194,4 +194,147 @@ def can_create_html_report
     end
   end
+
+  def can_create_gcov_html_report_from_crashing_test_runner_with_enabled_debug_and_cmd_args_set_to_true_for_test_cases_not_causing_crash
+    @c.with_context do
+      Dir.chdir @proj_name do
+        FileUtils.cp test_asset_path("example_file.h"), 'src/'
+        FileUtils.cp test_asset_path("example_file.c"), 'src/'
+        FileUtils.cp test_asset_path("test_example_file_sigsegv.c"), 'test/'
+        FileUtils.cp test_asset_path("project_with_guts_gcov.yml"), 'project.yml'
+
+        add_line = false
+        updated_prj_yml = []
+        File.read('project.yml').split("\n").each do |line|
+          if line =~ /\:project\:/
+            add_line = true
+            updated_prj_yml.append(line)
+          else
+            if add_line
+              updated_prj_yml.append('  :use_backtrace_gdb_reporter: TRUE')
+              add_line = false
+            end
+            updated_prj_yml.append(line)
+          end
+        end
+        enable_unity_extra_args = "\n:test_runner:\n"\
+                                  "  :cmdline_args: true\n"
+
+        updated_prj_yml.insert(updated_prj_yml.length() -1, enable_unity_extra_args)
+
+        File.write('project.yml', updated_prj_yml.join("\n"), mode: 'w')
+
+        output = `bundle exec ruby -S ceedling gcov:all 2>&1`
+        expect($?.exitstatus).to match(1) # Test should fail as sigsegv is called
+        gcov_html_report = `bundle exec ruby -S ceedling utils:gcov 2>&1`
+        expect($?.exitstatus).to match(0)
+        expect(output).to match(/Segmentation fault/i)
+        expect(output).to match(/Unit test failures./)
+        expect(File.exist?('./build/gcov/results/test_example_file_sigsegv.fail'))
+        output_rd = File.read('./build/gcov/results/test_example_file_sigsegv.fail')
+        expect(output_rd =~ /test_add_numbers_will_fail \(\) at test\/test_example_file_sigsegv.c\:14/ )
+        expect(output).to match(/TESTED:\s+2/)
+        expect(output).to match(/PASSED:\s+1/)
+        expect(output).to match(/FAILED:\s+1/)
+        expect(output).to match(/IGNORED:\s+0/)
+        expect(output).to match(/example_file.c Lines executed:50.00% of 4/)
+
+        expect(gcov_html_report).to match(/Creating gcov results report\(s\) in 'build\/artifacts\/gcov'\.\.\. Done/)
+        expect(File.exist?('build/artifacts/gcov/GcovCoverageResults.html')).to eq true
+      end
+    end
+  end
+
+  def can_create_gcov_html_report_from_crashing_test_runner_with_enabled_debug_and_cmd_args_set_to_true_with_zero_coverage
+    @c.with_context do
+      Dir.chdir @proj_name do
+        FileUtils.cp test_asset_path("example_file.h"), 'src/'
+        FileUtils.cp test_asset_path("example_file.c"), 'src/'
+        FileUtils.cp test_asset_path("test_example_file_sigsegv.c"), 'test/'
+        FileUtils.cp test_asset_path("project_with_guts_gcov.yml"), 'project.yml'
+
+        add_line = false
+        updated_prj_yml = []
+        File.read('project.yml').split("\n").each do |line|
```

Comment:
Hi @mvandervoord ,

May I suggest to change this part responsible by updating project.yml to look:

```
File.read('project.yml').split("\n").each do |line|
          if line =~ /\:use_backtrace_gdb_reporter\:/
            line = line.gsub("FALSE", "TRUE")
          end
          updated_prj_yml.append(line)
        end
```